contracts/payment/MCPayment.sol

```diff
@@ -339,6 +347,18 @@ contract MCPayment is Ownable2StepUpgradeable, EIP712Upgradeable {
         _withdraw(amount, owner());
     }
 
+    /**
+     * @dev Withdraw ERC-20 balance to owner
+     */
+    function ownerERC20Withdraw(address token) public onlyOwner {
+        uint amount = IERC20(token).balanceOf(address(this));
+        if (amount == 0) {
+            revert WithdrawError("There is no balance to withdraw");
```

Comment:
I would avoid adding strings as custom error params. It may cost more gas. Better change the error name